index.d.ts

```diff
@@ -1149,7 +1170,7 @@ declare module 'mongoose' {
     pre<T extends Model<DocType> = M>(method: 'insertMany' | RegExp, options: SchemaPreOptions, fn: (this: T, next: (err: CallbackError) => void) => void): this;
 
     /** Object of currently defined query helpers on this schema. */
-    query: any;
+    query: TQueryHelpers;
```

Comment:
Ideally this should fall back to `any` if `TQueryHelpers` isn't set

Reply:
i made it fall back to `Record<string, any>` would be better, if you agree.